### ISSUE TITLE
[Issue]: Episode date incorrect (web client)

### ISSUE THREAD

**Issue by TheHYPO35** [jellyfin/jellyfin#11811]
### Please describe your bug

The air date listed on an episode page is not the correct date for some (but not all) shows. The metadata for the episode is correct but the info page screen shows the wrong date.

This was an issue on the Firestick/Android client last year or earlier this year that I filed a bug report for and it was resolved, and at the time, the web client was fine, but now the web client has developed this issue. In the Android client's case, the issue (as I understand it) was that the relevant field was using the local time zone offset, but since the airdate is a fixed metadata field, it really shouldn't be adjusted by time zones at all. It should be the absolute date set in the metadata.

Here is the bug report for that https://github.com/jellyfin/jellyfin-androidtv/issues/3190

I actually just realized that the episodes that show the correct date seem to be the episodes that premiered from fall to spring during daylight savings time, but not the episodes from March to October, so again, it sounds like potentially a time zone offset issue. It may also be specific to certain time zones where the offset makes this happen. I'm in GMT-5 

### Reproduction Steps

Open the web browser client or the windows Jellyfin Media Player client. Open the info page for an episode of a TV show (Particularly, check an episode that aired between November and March.). Compare the date listed for the episode at the top to the air date on the metadata page.

### Jellyfin Version

10.9.0

### Environment

```markdown
- OS: Debian 12
- Linux Kernel: 6.6.28+rpt-rpi-v8
- Virtualization:
- Clients:  Web version 10.9.2 (as well as Jellyfin Media Player 10.9.2)
- Browser: Chrome

I don't believe the processor/playback values are likely to be relevant to this bug.
```

### Jellyfin logs

```shell
n/a
```

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by NotSaifA** [comment]
This is a [jellyfin-web](https://github.com/jellyfin/jellyfin-web) issue, was able to replicate.

**Comment by NotSaifA** [comment]
See https://github.com/jellyfin/jellyfin-web/issues/369

**Comment by TimGels** [comment]
@TheHYPO35 Can you re-test this issue in 10.9.3? Otherwise I will proceed to close this issue as solved.

**Comment by TheHYPO35** [comment]
@TimGels  Retested. Browser webversion 10.9.3 displays the correct date. Jellyfin Media Player which is still 10.9.2 displays the incorrect date, so it appears to be fixed in .3

**Comment by TimGels** [comment]
Fixed by [#5599](https://github.com/jellyfin/jellyfin-web/pull/5599) in 10.9.3.

**Comment by TimGels** [comment]
Fixed by https://github.com/jellyfin/jellyfin-web/pull/5599 in 10.9.3.
